shapes/neurosciencegraph/commons/annotation/schema.json

```diff
@@ -146,6 +151,21 @@
         }
       ]
     },
+    {
+      "@id": "this:HasBodyShape",
+      "@type": "sh:NodeShape",
+      "class": "nsg:AnnotationBody",
```

Comment:
I recommend to keep the class constraint on the property shape with "path": "nsg:hasBody" like this:

```
{
              "path": "nsg:hasBody",
              "name": "Body",
              "description": "The entity or value that the target entity is being annotated with.",
              "class": "nsg:AnnotationBody",
	      "node": "this:HasBodyShape"
}
```

The class can then be removed from the shape this:HasBodyShape